src/NewHoursReport.java

```diff
@@ -1,131 +1,487 @@
+import java.util.List;
+
 import javax.swing.*;
 import javax.swing.filechooser.FileNameExtensionFilter;
 
-import org.apache.poi.*;
-import org.apache.poi.openxml4j.exceptions.InvalidFormatException;
+import org.apache.poi.hssf.usermodel.HSSFDateUtil;
 import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellStyle;
+import org.apache.poi.ss.usermodel.CellType;
 import org.apache.poi.ss.usermodel.CreationHelper;
+import org.apache.poi.ss.usermodel.DataConsolidateFunction;
 import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.xssf.usermodel.XSSFCell;
+import org.apache.poi.xssf.usermodel.XSSFRow;
 import org.apache.poi.xssf.usermodel.XSSFSheet;
 import org.apache.poi.xssf.usermodel.XSSFWorkbook;
 
 
 
-public class NewHoursReport extends JPanel  {
-	JFrame frame;
-	JLabel label;
+
+public class NewHoursReport extends JFrame  {
+	  static JFrame frame;
+	 JLabel label;
+	 Date date;
+	 JFrame dateFrame;
+	 JButton homeBut;
+	 JButton nextBut;
+	 JButton prevBut;
+	 JButton browse;
+	 JButton exit;
+	 JPanel panelContainer;
+	 JFileChooser fileChooser;
+	 static JTextField startDate;
+	 static JTextField endDate;
+
+	 public static final String OUTPUT_FILE = "Hours_" + getCurrentTimeStamp() +
+				".xlsx";
+
+	static String[] header = {"Month", "Customer", "Project", "Hours", "Cost", 
+			"Invoiced", "Profitability"};
+	static String[] dateHeader = {" ","Jan","Feb","Mar","Apr", "May","Jun","Jul",
+			"Aug","Sep","Oct","Nov","Dec"};
 	
+	protected static ArrayList<Object> input = new ArrayList<>();
+	protected static ArrayList<Object> projects = new ArrayList<>();
+	protected static ArrayList<Object> dates = new ArrayList<>();
+	protected static ArrayList<Object> billStatus = new ArrayList<>();
+	protected static ArrayList<Object> names = new ArrayList<>();
+	protected static ArrayList<Object> duration = new ArrayList<>();
+	protected static ArrayList<Object> contractServ = new ArrayList<>();
 	
-	public static void main(String[] args) {
-		// Just used for testing 
-		JFrame frame = new JFrame();
-		GUI gui = new GUI(frame);
-		
+	
+		public static void main(String[] args) throws IOException {
+			 
+			
+			javax.swing.SwingUtilities.invokeLater(new Runnable() {
+				public void run() {
+					createAndShowFrame();
+				}
+			});
+				
+		}
 		
-	}
+		// next 4 methods set up the GUI
+		public static JFrame createAndShowFrame() {
+	 		
+	 		frame = new JFrame();
+	 		
+	 		JLabel label = new JLabel("Please select a file to open");
+	 		
+	 		//create the panel to add it to
+	 		JPanel topPane = new JPanel();
+	 		JPanel bottomPane = new JPanel();
+	 		
+	 		//create 2 buttons Browse and Cancel
+	 		JButton browse = createButton(40,40,"Browse");
+	 		JButton cancel = createButton(40,40,"Cancel");
+	 		browseListenerClass listener1 = new browseListenerClass();
+	 		cancelListenerClass listener2 = new cancelListenerClass();
+	 		browse.addActionListener(listener1);
+	 		cancel.addActionListener(listener2);
+	 		
```

Comment:
Remove extra spaces. One line is fine for distinguishing sections of code but limit it to that.